Assets/MRTK/SDK/Features/UX/Scripts/Cursors/TeleportCursor.cs

```diff
@@ -114,6 +114,11 @@ protected override void UpdateCursorTransform()
             arrowTransform.eulerAngles = cursorOrientation;
         }
 
+        public override void SetVisibility(bool visible)
+        {
+            this.PrimaryCursorVisual.gameObject.SetChildrenActive(visible);
```

Comment:
Does this need to call into `base.SetVisibility`?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/ba230dee9f8a61dee35be89d2570317339f3d234/Assets/MRTK/SDK/Features/UX/Scripts/Cursors/BaseCursor.cs#L216-L223

Reply:
nope the underlying `base.SetVisibility` is what caused a bug where the teleport cursor would suddenly disappear/reappear when using hand constraints. This is because the base class's implementation would deactivate the gameobject with the animator, causing the animation state to get reset and for the disappearing animation to play unnecessarily.